extensions/wikia/AdEngine/js/slot/floatingMedrec.js

```diff
@@ -32,10 +36,17 @@ define('ext.wikia.adEngine.slot.floatingMedrec', [
 			$adSlot = $('<div class="wikia-ad"></div>').attr('id', slotName),
 			$footer = $('#WikiaFooter'),
 			$placeHolder = $('#WikiaAdInContentPlaceHolder'),
-			$win = $(win);
+			$win = $(win),
+			refresh = {
+				refreshAdPos: 0,
+				lastRefreshTime: new Date(),
+				refreshNumber: 0
+			};
 
 		function getStartPosition(placeHolder) {
-			return parseInt(placeHolder.offset().top, 10) - globalNavigationHeight - margin;
+			return parseInt(placeHolder.offset().top, 10) -
+					// TODO understand when palceholder height is required
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Complete the task associated to this TODO comment. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AS1135)